modules/plugins/ez_plugins/machinery.py

```diff
@@ -0,0 +1,13 @@
+from ....machinery.installer import IPluginInstaller, PluginInstallerInfo, PluginInstallationResult
+from ....machinery.loader import IPluginLoader, PluginLoaderInfo
+from ....plugin_info import PluginInfo
```

Comment:
What's with all this relative imports? Does it have to be relative?

Reply:
Fixed